src/main/java/org/broadinstitute/dsm/cf/TestBostonKitTrackerDispatcher.java

```diff
@@ -74,7 +74,6 @@ public void accept(PubsubMessage pubsubMessage, Context context) throws Exceptio
                 " ))";
         String SQL_AVOID_DELIVERED = " and (tracking_to_id is not null or tracking_return_id is not null ) and kit.test_result is null " +
                 " and ( ups_status_description is null or ups_status_description not like \"%Delivered%\") "+
-                " and  from_unixtime(created_date/1000) > NOW() - INTERVAL 180 DAY"+
```

Comment:
If you run this query with and without the 180 day change, what does the difference look like?  Will anything break with a higher volume of stuff as a result?

Reply:
The size of the result changes from ~7000 kits to ~13000 kits. We run the CF in different instances each with 1000 kits running parallel wise, and this query is also getting the kits in batches of that size. So I don't think we will face a memory issue.
I see that the execution time is around 1.5 min now and I have allocated 540 seconds so I don't think it will break by that.
However, if there are only 3 or 4 kits showing up everyday that are not ordered in CE, I am not sure it is worth checking all the 13k kits for them. especially since their ups label is expired and we will get lots of nulls and errors from UPS.
I can make a quick CF that takes the kit label, checks for the info manually ( collection time, external order number, short id) and orders it so we do ordering less manually.

Reply:
How about we change this to 365 days instead of completely removing it?